pages/embed/contribution-flow.js

```diff
@@ -149,7 +149,7 @@ class NewContributionFlowPage extends React.Component {
     if (contributionBlocker) {
       return <ContributionBlocker blocker={contributionBlocker} account={account} />;
     } else if (step === 'success') {
-      return <ContributionFlowSuccess collective={account} isEmbed />;
+      return <ContributionFlowSuccess collective={account} isCrypto={this.props.verb === 'crypto'} isEmbed />;
```

Comment:
Still using `verb` here?

Could you double check **everywhere**?

Reply:
Ops sorry, something I've missed; double checked and this is the only place that I've missed it seems. Corrected it.